src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.DebugVerifier.cs

```diff
@@ -187,6 +187,20 @@ private void VisitBinaryOperatorChildren(BoundBinaryOperatorBase node)
                 Visit(node.BoundContainingTypeOpt);
                 return null;
             }
+
+            public override BoundNode? VisitWithExpression(BoundWithExpression node)
+            {
+                Visit(node.Receiver);
+                // PROTOTYPE: We shouldn't need to implement this method manually here.
```

Comment:
Wary about adding more PROTOTYPE debt here. Seems more efficient to just file an issue and reference it here. 

Keep the PROTOTYYPE though if you expect a change you're already making in next 10 days will remove it.

Reply:
Looks like #44582 will fix this by changing the bound node.